buf.yaml

```diff
@@ -1,20 +1,20 @@
 # buf.yaml files should always be in the root of your repository.
 # More details can be found on our wiki page {@link 'https://github.com/oppia/oppia-android/wiki/Buf-Guide'
+
+version: v1beta1
+
 build:
   roots:
     # Roots is the directories that all .proto files are contained within, and
     # all imports must derive from.
     # Only *.proto files within these directory will be compiled.
-    - .
-  # Excludes is the list of directories within root to exclude.
-  excludes:
```

Comment:
This approach actually seems more flexible since we'll be introducing package-specific proto files in the future. Shouldn't we default to including all protos?

Reply:
> Your directory structure was likely always broken if checking lint errors in all files.

https://github.com/bufbuild/buf/issues/273#issuecomment-785868496